.github/workflows/main.yml

```
@@ -107,8 +107,8 @@ jobs:
         uses: actions/setup-python@v2
         with:
           python-version: 3.9
-      - name: Install GDAL
-        run: sudo apt-get update && sudo apt install gdal-bin 
+      - name: Install GDAL and gettext
+        run: sudo apt-get update && sudo apt install gdal-bin && sudo apt install gettext
```

Nit-picking, but why not:
```suggestion
        run: sudo apt-get update && sudo apt install gdal-bin gettext
```